README.md

```diff
@@ -50,11 +50,10 @@ Currently the methods are heavily commented in [the source code](./index.js), so
 
 ### Setup
 
-- Install [Node.js](https://nodejs.org) version 12
+- Install [Node.js](https://nodejs.org) version 14
```

Comment:
This change was unrelated; I just happened to notice that it was wrong. We forgot this when the Node.js minimum version was bumped to v14.